x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/es_search_source.js

```diff
@@ -228,9 +245,16 @@ export class ESSearchSource extends AbstractESSource {
     };
   }
 
+  // Returns true when top hits is enabled and properly configured
   _isTopHits() {
-    const { useTopHits, topHitsSplitField, topHitsTimeField } = this._descriptor;
-    return !!(useTopHits && topHitsSplitField && topHitsTimeField);
+    const { useTopHits, topHitsSplitField } = this._descriptor;
+    return !!(useTopHits && topHitsSplitField);
+  }
+
+  // Returns true when sort is properly configured
+  _hasSort() {
+    const { sortField, sortOrder } = this._descriptor;
+    return !!(sortField && sortOrder);
```

Comment:
that coercion to a booleon looks so funky.

either do ``return !!sortField && !!sortOrder`` or remove double negation altogether